README.md

```diff
@@ -1,79 +1,134 @@
 # Hyperdash Python SDK
 
-The Hyperdash Python SDK is the official SDK for [Hyperdash.io](https://hyperdash.io). Once installed, the SDK automatically monitors your machine learning jobs.
+[Hyperdash](https://hyperdash.io) is a machine learning monitoring library, written in Python and capable of running alongside Tensorflow, Scikit-Learn, and other modelling libraries. Hyperdash provides visualizations similar to Tensorboard. It was developed with a focus on enabling fast knowledge gain.
 
-## Installation
+Use Hyperdash if you need model monitoring that:
 
-`pip install --upgrade pip && pip install hyperdash`
+* Is fast and easy to setup and use.
+* Tracks your hyperparmeters across different model experiments.
+* Graphs performance metrics (loss, reward, etc.) in real-time.
```

Comment:
Should we say tracks instead?

Reply:
I think it's implied, and I'd rather focus on the visual of real-time graphs.